avtozip/store/views.py

```diff
@@ -1,8 +1,26 @@
 """Store application views."""
 
+from django.http import HttpResponseRedirect
 from django.shortcuts import render
 
+from .api.resources import ProductResource
+from .forms import ProductFormSet
+
 
 def index_view(request):
     """INDEX view of store application."""
-    return render(request, 'store/index.html')
+    if request.method == 'POST':
+        formset = ProductFormSet(request.POST)
+        if formset.is_valid():
+            formset.save()
+            return HttpResponseRedirect(request.META.get('HTTP_REFERER'))
+    else:
+        res = ProductResource()
+        request_bundle = res.build_bundle(request=request)
+        formset = ProductFormSet(queryset=res.obj_get_list(request_bundle))
+
+    context = {
+        'formset_headers': formset.forms[-1].visible_fields,
```

Comment:
We do not need this additional context variable.
Please look [here](http://stackoverflow.com/questions/15849240/custom-formset-templates-in-django) to find out template tags like ` {% if forloop.first %}`